src/consensus/aft/raft.h

```diff
@@ -74,6 +74,7 @@ namespace aft
 
     ReplicaState replica_state;
     std::chrono::milliseconds timeout_elapsed;
+    kv::Version election_index = 0;
```

Comment:
Can you add a comment explaining what this index is?

Reply:
Sure